nni/compression/pytorch/utils/counter.py

```diff
@@ -59,7 +60,13 @@ def __init__(self, custom_ops=None, mode='default'):
                 nn.AdaptiveAvgPool3d: self._count_adap_avgpool,
                 nn.Upsample: self._count_upsample,
                 nn.UpsamplingBilinear2d: self._count_upsample,
-                nn.UpsamplingNearest2d: self._count_upsample
+                nn.UpsamplingNearest2d: self._count_upsample,
+                nn.RNNCell: self._count_rnn_cell,
+                nn.GRUCell: self._count_gru_cell,
+                nn.LSTMCell: self._count_lstm_cell,
+                nn.RNN: self._count_rnn,
+                nn.GRU: self._count_gru,
+                nn.LSTM: self._count_lstm,
```

Comment:
just curious, why only `conv` and `linear` are supported by default?

Reply:
Most pruning papers target on the classification task and only report results of `conv` and `linear`. Move RNN modules to default would be better.